common/client_config_v2.go

```diff
@@ -24,6 +26,13 @@ type ClientConfigV2 struct {
 	PutTries                   int
 	MaxBlobSizeBytes           uint64
 	EigenDACertVerifierAddress string
+
+	// Fields required for validator payload retrieval
+	BLSOperatorStateRetrieverAddr string
+	EigenDAServiceManagerAddr     string
```

Comment:
if they are required for validator payload retrieval, why aren't they in the ValidatorPayloadRetrieverCfg?

Reply:
the `ValidatorPayloadRetrieverCfg` only contains values directly needed by the `ValidatorPayloadRetriever`.

These addresses are needed by the `RetrievalClient`, which is passed into the `ValidatorPayloadRetriever` already constructed.

The problem is that there is no `RetrievalClientConfig`: the `RetrievalClient` just accepts all of its required parameters directly. Ideally, a `RetrievalClientConfig` would be created that contains all necessary values for construction, but this would require core changes.